policies_folder_status.py

```diff
@@ -234,3 +236,30 @@ def post_status_transition(ctx: rule.Context,
         submitter = folder.get_submitter(ctx, path)
         message = "Data package rejected for vault"
         notifications.set(ctx, actor, submitter, path, message)
+
+
+def delete_research_package(ctx: rule.Context, path: str) -> None:
+    """Delete research package collection during copy_to_vault, if AVU constants.KEEP_RESEARCH_COPY is False
+
+    :param ctx:  Combined type of a callback and rei struct
+    :param path: Collection path in research zone
+    """
+    try:
+        # Get AVU to decide keep copy or delete research package
+        keep_package = avu.get_attr_val_of_coll(ctx, path, constants.KEEP_RESEARCH_COPY)
+
+        # Delete collection if KEEP_RESEARCH_COPY is "False"
+        if keep_package == "False":
```

Comment:
We can write this as `if not keep_package:`.

Might be more intuitive if we name the variable delete_research_copy, because that is the happy flow of this condition.